docs/sources/variables/filter-variables-with-regex.md

```diff
@@ -79,4 +79,30 @@ demo.robustperception.io:9090
 demo.robustperception.io:9093
 demo.robustperception.io:9100
 ```
+
+## Filter and use named text and value capture groups
+
+This can be useful if the display text should differ from the value.
+
+List of options:
```

Comment:
I am confused. Are these options an example, or are they common to all users? This section needs more context, as do the Regex and Result below.

Reply:
Sure. I was following the format of the examples earlier the file. Agree that it could be improved - documentation is not my strong point ;) I’ll rework with expanded examples that relate tot the use case in the issue that it fixes